Tchap/Modules/Favourites/FavouriteMessages/FavouriteMessagesViewAction.swift

```diff
@@ -22,5 +22,6 @@ import Foundation
 enum FavouriteMessagesViewAction {
     case loadData
     case tapEvent(roomId: String, eventId: String)
+    case tapAction(fragment: String)
```

Comment:
Please rename this case. I suggest "handlePermalinkFragment" or something similar